tools/topology/topology1/CMakeLists.txt

```diff
@@ -164,6 +164,7 @@ set(TPLGS
 	"sof-tgl-max98357a-rt5682\;sof-adl-max98360a-rt5682-2way\;-DCODEC=MAX98360A_TDM\;-DFMT=s32le\;-DPLATFORM=adl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-D2CH_2WAY\;-DBT_OFFLOAD"
 	"sof-tgl-max98357a-rt5682\;sof-adl-max98360a-rt5682-4ch\;-DCODEC=MAX98360A_TDM\;-DFMT=s32le\;-DPLATFORM=adl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-D4CH_PASSTHROUGH\;-DBT_OFFLOAD"
 	"sof-tgl-max98357a-rt5682\;sof-adl-max98360a-rt5682-rtnr\;-DCODEC=MAX98360A\;-DFMT=s32le\;-DPLATFORM=adl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-DCHANNELS=2\;-DRTNR"
+	"sof-tgl-max98357a-rt5682\;sof-adl-max98360a-rt5682-rtnr-google-aec\;-DCODEC=MAX98360A\;-DFMT=s32le\;-DPLATFORM=adl\;-DLINUX_MACHINE_DRIVER=sof_rt5682\;-DAMP_SSP=1\;-DCHANNELS=2\;-DGOOGLE_RTC_AUDIO_PROCESSING\;-DRTNR\;-DNOHOTWORD\;-DNO16KDMIC"
```

Comment:
@johnylin76 looks good but do these need to go into main? OR can this go into the adl release branch?

Reply:
@ranj063 does it not need to go in both, or would the build break if topologies that use 3P modules ? Fwiw, having production topologies in main would at least make sure they all build in CI.

Reply:
@ranj063 is there a case where things should not land on main first?

Reply:
It's be rather odd to have topologies in the SOF tree that cannot be used with the firmware generated from the SOF tree. I can't recall having seen such a precedent.

Reply:
ack - but there is no harm ikeeping topologies that use 3P algos upstream. They can still be used as templates for others integrating similar topologies or 3P processing.

Reply:
agree, we already have way too many topologies that were added "just in case". let's be clear and move those to a separate directory.